templates/common/on-prem/files/configure-ip-forwarding.yaml

```diff
@@ -0,0 +1,48 @@
+mode: 0755
+path: "/usr/local/bin/configure-ip-forwarding.sh"
+contents:
+  inline: |
+    #!/bin/bash
+    set -ex
+    
+    echo "Configuring IP Forwarding..."
+    ovnk_config_dir='/etc/ovnk'
+    ip_hint_file="/run/nodeip-configuration/primary-ip"
+    
+    get_ip_from_ip_hint_file() {
+      local ip_hint_file="$1"
+      if [[ ! -f "${ip_hint_file}" ]]; then
+        return
+      fi
+      ip_hint=$(cat "${ip_hint_file}")
+      echo "${ip_hint}"
+    }
+    
+    get_nodeip_hint_interface() {
+      local ip_hint=""
+      local ip_hint_file="$1"
+      local iface=""
+
+      ip_hint=$(get_ip_from_ip_hint_file "${ip_hint_file}")
+      if [[ -z "${ip_hint}"  ]]; then
+        return
+      fi
+
+      iface=$(ip -j addr | jq -r "first(.[] | select(any(.addr_info[]; .local==\"${ip_hint}\"))) | .ifname")
+      if [[ -n "${iface}" ]]; then
+        echo "${iface}"
+      fi
+    }
+    
+    iface=$(get_nodeip_hint_interface "${ip_hint_file}")
```

Comment:
the `ip_hint_file` seems global? do we want to pass this into each function specifically?

Reply:
i copied this from configure-ovs so was just using the same functions/format

Reply:
ack not a biggie, just thought can avoid the extra $1 checks, but its ok the way it is functionally speaking..